xr_sample_project/Source/xr_samplesproject/XRTableTop/XRTabletopComponent.cpp

```diff
@@ -126,7 +126,8 @@ void UXRTabletopComponent::PreUpdateTabletop()
 
 		if (bExtentChanged)
 		{
-			MapComponent->SetOriginPosition(newExtent.ExtentCenter);
+			UArcGISPoint* ExtentCenter = UArcGISPoint::CreateArcGISPointWithXY(newExtent.ExtentCenter.X, newExtent.ExtentCenter.Y);
+			MapComponent->SetOriginPosition(ExtentCenter);
```

Comment:
```suggestion
			MapComponent->SetOriginPosition(CenterPosition);
```
If you make this change, you can move the definition of `newExtent` to the else-block below (or directly pass `{ Esri::ArcGISMapsSDK::Utils::ToInstanceData(CenterPosition), Shape, ExtentDimensions}` to `SetExtent` below ).